Backend/src/main/java/com/example/acmeplex/paymentsystem/service/PaymentService.java

```diff
@@ -0,0 +1,178 @@
+package com.example.acmeplex.paymentsystem.service;
+
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Date;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.example.acmeplex.paymentsystem.repository.CreditRecordRepository;
+import com.example.acmeplex.paymentsystem.repository.PaymentRepository;
+import com.example.acmeplex.usersystem.service.RegisteredUserService;
+import com.example.acmeplex.moviesystem.repository.TicketRepository;
+import com.example.acmeplex.paymentsystem.entity.CreditRecord;
+import com.example.acmeplex.paymentsystem.entity.Payment;
+
+import jakarta.transaction.Transactional;
```

Comment:
We should use `import org.springframework.transaction.annotation.Transactional;` in springboot